qiskit/algorithms/eigensolvers/vqd.py

```diff
@@ -160,12 +165,12 @@ def __init__(
         self._eval_count = 0
 
     @property
-    def initial_point(self) -> Sequence[float] | None:
+    def initial_point(self) -> Optional[Union[np.ndarray, List[np.ndarray]]]:
```

Comment:
Could we keep the previous way of documenting? That would mean this here is 
```
Sequence[float] | Sequence[Sequence[float]] | None
```

Reply:
You can directly apply this suggestion:

```suggestion
    def initial_point(self) -> Sequence[float] | Sequence[Sequence[float]] | None:
```